### DISCUSSION TITLE
Chart axis label color isn't editable

### DISCUSSION THREAD

**Discussion by msullivan-NoVa** [GSI-CS-CO/chart-fx#486] [Q&A] [answered]
I'm unable to change the axis label text color when using a dark theme, it remains black. I've looked at the 'financial' example charts that show the charts using 'dark' theme and they appear to show the same thing. In CSS I've tried `.axis-label { -fx-text-fill: white; }` but it has not affect, the axis label remains black. Is anyone aware of a work around?

**Comment by wirew0rm** [comment, marked as answer]
Hey, just use `.axis-label { -fx-fill: white; }`. The reason is, that the Label is not rendered as a javafx text label but directly rendered to the graphics context where the paint style from the normal fill property is used. I have not investigated if it would be possible or desirable to change this.

**Reply by msullivan-NoVa** [reply]
Worked, thanks
